components/omega/doc/design/VerticalMixingCoeff.md

```diff
@@ -0,0 +1,184 @@
+
+### 2.3 Requirement: Vertical mixing models cannot ingest single columns at a time
+
+Many standard vertical mixing libraries (e.g. CVMix) receive and operate on one column at a time. This results in poor performance, especially on accelerators. Any utilized model of vertical mixing must ingest and operate on multiple columns concurrently.
+
+## 3 Algorithmic Formulation
+
+For Omega-1 a few simple vertical mixing algorithms will be used. When computing $\kappa$ and $\nu$, linear superposition when combining the various parameterizations is assumed.
+
+### 3.1 Richardson number dependent mixing
+
+One of the simplest class of models of vertical turbulence are polynomial functions of the gradient Richardson number. As in MPAS-Ocean, we choose to define the Richardson number as
+
+$$
+Ri = \frac{N^2}{\left|\frac{\partial \mathbf{U}}{\partial z}\right|^2}\,,
+$$
+
+where $N^2$ is the Brunt Vaisala Frequency, which for non Boussinesq flows is defined as
+
+$$
+N^2 = \frac{g}{\rho_0}\frac{\partial \rho}{\partial z}\,.
+$$
+
+If we assume this term is defined at the top of the cell, it is discretized as
+
+$$
+N^2(k) = g \rho_0 \frac{\rho_{DD}(k) - \rho(k)}{z_m(k-1) - z_m(k)}\,.
+$$
+
+Here $\rho_{DD}$ is the density of the fluid of layer k-1 displaced to layer k adiabatically.
+
+The shear in the denominator, also defined at the top of the cell, is discretized as
+
+$$
+\left| \frac{\partial \mathbf{U}}{\partial z}\right|^2 = \left(\frac{U_n(k-1)-U_n(k)}{z_m(k-1) - z_m(k)}\right)^2 + \left(\frac{U_t(k-1)-U_t(k)}{z_m(k-1) - z_m(k)}\right)^2\,.
+$$
+
+Where the subscripts *n* and *t* are the normal and tangential velocities respectively.
+
+With the definition of the gradient richardson number, the viscosity and diffusivity are defined according to [Pacanowski and Philander (1981)](https://journals.ametsoc.org/view/journals/phoc/11/11/1520-0485_1981_011_1443_povmin_2_0_co_2.xml?tab_body=pdf) as
+
+$$
+\nu = \frac{\nu_o}{(1+\alpha Ri)^n} + \nu_b\,,
+$$
+
+$$
+\kappa = \frac{\nu}{(1+\alpha Ri)} + \kappa_b\,.
+$$
+
+$\nu$ and $\kappa$ are defined at the top of the cell, similarly to $Ri$ and $N^2$. In these formulae, $\alpha$ and *n* are tunable parameters, most often chosen as 5 and 2 respectively. $\nu_b$ and $\kappa_b$ are the background viscosity and diffusivity respectively. MPAS-Ocean has an additional $\kappa_{b,passive}$ for applying to just the passive tracers, leaving the active tracers with $\kappa_b$.
+
+### 3.2 Convective Instability Mixing
+
+Commonly (and how this is currently handled by CVMix in MPAS-O), mixing due to convective instability is treated as a step function for the diffusivity and viscosity. This is often represented as
+
+$$
+\kappa =
+\begin{cases}
+\kappa_{conv} \quad \text{ if } N^2 \leq N^2_{crit}\\
+0 \quad \text{ if } N^2 > N^2_{crit}
```

Comment:
```suggestion
\kappa_{b} + \kappa_{conv} \quad \text{ if } N^2 \leq N^2_{crit}\\
\kappa_{b} \quad \text{ if } N^2 > N^2_{crit}
```

I think this is more consistent with your notation for Ri-dependent mixing, which also has the background values added